src/mpc/structs.jl

```diff
@@ -242,6 +242,42 @@ function MPCElectricTariff(d::Dict)
 end
 
 
+"""
+MPCElectricUtility
```

Comment:
If we had defaults of zeros for all the new `ElectricUtility` fields would we not need a separate `MPCElectricUtility`? It would be great for maintainability if we could reuse as many structs as possible in the MPC code.

Reply:
In the non-MPC case, if the emissions inputs aren't provided then AVERT data is used. In the MPC case, we don't want to do that lookup, and can't anyways because there is no lat/long. Best way of eliminating the MPCElectricUtility that I can think of at the moment would be to add nothing defaults for the lat/long inputs, and in that case don't look up AVERT data. It could theoretically lead to misuse of the constructor without lat/long in the non MPC case, but I think that risk is better than having to maintain separate structs.